nautobot/core/views/viewsets.py

```diff
@@ -0,0 +1,15 @@
+from . import mixins
+
+
+class NautobotUIViewSet(
+    mixins.ObjectDetailViewMixin,
+    mixins.ObjectListViewMixin,
+    mixins.ObjectEditViewMixin,
+    mixins.ObjectDestroyViewMixin,
+    mixins.ObjectBulkDestroyViewMixin,
+    mixins.ObjectBulkCreateViewMixin,
+    mixins.ObjectBulkUpdateViewMixin,
+):
+    """
+    This is the UI BaseViewSet you should inherit.
```

Comment:
Please revise this to say something a little more descriptive. For example the DRF `ModelViewSet` says:

```
    """
    A viewset that provides default `create()`, `retrieve()`, `update()`,
    `partial_update()`, `destroy()` and `list()` actions.
    """
```

Would be good to also mention that this is meant for UI views.